bugbug/repository.py

```diff
@@ -32,6 +33,24 @@
 EXPERIENCE_TIMESPAN = 90
 EXPERIENCE_TIMESPAN_TEXT = f"{EXPERIENCE_TIMESPAN}_days"
 
+FILE_TYPES = {
```

Comment:
Nit: the mapping could be inverted to have a smaller and more readable dict (FileType -> Extension). Then you can define another constant which inverts the inverted mapping